src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -178,6 +180,12 @@ export async function startTransformByQ() {
             await vscode.commands.executeCommand('aws.amazonq.refresh') // so that button updates
             uploadId = await uploadPayload(payloadFileName)
         } catch (error) {
+            telemetry.codeTransform_logGeneralError.emit({
+                codeTransformSessionId: codeTransformTelemetryState.getSessionId(),
+                codeTransformApiErrorMessage: error instanceof Error ? error.message : errorMessage,
```

Comment:
`error.message` may include the pre-signed upload URL which is not allowed per AppSec

Reply:
Do you know if IntelliJ might have a similar issue? It's logging the full error message at the moment, which doesn't seem to contain pre-signed url per kibana records: https://github.com/aws/aws-toolkit-jetbrains/blob/9fd13eb1a8dd9b291a430327e3aba6f2614d610e/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerSession.kt#L99. We can unified the experience and use static message only?

Reply:
Hmm interesting. Yes seems like IntelliJ then has the same issue which would be here: https://github.com/aws/aws-toolkit-jetbrains/blob/9fd13eb1a8dd9b291a430327e3aba6f2614d610e/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerSession.kt#L326. I'm not sure how often (if ever) the pre-signed URL will be included in the error message. It would be nice to have the full error message, but we should probably just use a static message. I'm pretty sure AppSec doesn't want to log or emit the pre-signed URL in any way.

Reply:
I updated `logGeneralError` to use static message for now